src/python/grpcio/commands.py

```diff
@@ -253,6 +253,10 @@ class BuildExt(build_ext.build_ext):
     LINK_OPTIONS = {}
 
     def build_extensions(self):
+        # This special conditioning is here due to difference of compiler
```

Comment:
I'm not sure I fully follow the reasoning presented in this comment. It sounds like this comment describes a difference in behavior between gcc and clang, but the special condition here (`if "darwin" in sys.platform`) is distinguishing mac from other systems. Could you clarify why this explains the special casing of mac builds? Would a simpler statement like "Darwin builds use a clang toolchain, while other systems use gcc" convey the same information?

Reply:
Default situation explained, thank you for pointing it out ;)